sopel/modules/wikipedia.py

```diff
@@ -262,3 +275,48 @@ def wikipedia(bot, trigger):
     else:
         query = query[0]
     say_snippet(bot, trigger, server, query)
+
+
+@commands('wplang')
+@example('.wplang pl')
+def wplang(bot, trigger):
+    if not trigger.group(3):
+        bot.reply(
+            "Your current Wikipedia language is: {}"
+            .format(
+                bot.db.get_nick_value(
+                    trigger.nick, 'wikipedia_lang',
+                    bot.config.wikipedia.default_lang)
+            )
+        )
+    else:
+        bot.db.set_nick_value(trigger.nick, 'wikipedia_lang', trigger.group(3))
```

Comment:
Not even a check on the validity of the lang here?

Reply:
Considered it.
Partially implemented it.
Realized that there's no nice way of making a regex strict enough to be useful, and I don't want to hard-code a list of valid language codes. (1. There are a lot of them. 2. Wikimedia might add more at any time.)
Deleted it. 😹

About all we could do is verify that the lang contains only a-z and hyphen, which is not particularly helpful. The user will figure out very soon if they set it wrong anyway, and it's easily changed again.

Reply:
Yeah, we could use https://pypi.org/project/pycountry/ at some point, but I think we can leave that for the future.

Reply:
Note for the future: `pycountry` language lookup won't recognize `simple`, `zh-min-nan`, `roa-tara`, or any of several other [perfectly valid](https://meta.wikimedia.org/wiki/List_of_Wikipedias) Wikipedia subdomains.